MekHQ/src/mekhq/campaign/parts/MissingAeroSensor.java

```diff
@@ -137,4 +137,18 @@ public TechAdvancement getTechAdvancement() {
     public PartRepairType getMRMSOptionType() {
         return PartRepairType.ELECTRONICS;
     }
+
+    // This function is over-ridden from it's parent because Aero Sensors depend on tonnage for small craft, but not for dropships/warships
```

Comment:
What would happen if we included this as JavaDocs, instead of comments?

Reply:
Comments been changed to be javadoc compatible, although I can't view the JavaDocs on my end so verifying the formatting is right will be appreciated.